src/api/Metadata.js

```diff
@@ -720,6 +720,71 @@ class Metadata extends File {
         }
     }
 
+    /**
+     * API for patching metadata on file
+     *
+     * @param {BoxItem} file - File object for which we are changing the description
+     * @param {Object} templateInstance - Metadata template instance
+     * @param {Array} operations - Array of JSON patch operations
+     * @param {Function} successCallback - Success callback
+     * @param {Function} errorCallback - Error callback
+     * @return {Promise}
+     */
+    async updateMetadataRedesigned(
```

Comment:
we need to settle on the phrase Redesigned or Redesign because im seeing both terms being used in other PRs
i.e. I see `createMetadataRedesign` and with this, it becomes very inconsistent

Reply:
Thanks! Renamed it to `updateMetadataRedesign` to keep consistent 👍